docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -87,11 +91,11 @@ Passkeys are associated with the domain they are created on, and cannot be used
 
 For example, a passkey created in `localhost` will be associated with the `localhost` domain, and therefore, will not work with your app's [hosted Account Portal or sign-in pages](/docs/customization/account-portal//overview#account-portal-overview).
 
-This will not be a problem for production instances, since the production Account Portal is a subdomain of your production app's domain.
+This won't affect production instances, as the production Account Portal is a subdomain of your app's domain.
 
 ### One-time password (OTP)
 
-When one of the OTP options are selected as an authentication strategy, users will receive a one-time code that they can use to complete the authentication process. The benefit of one-time codes is that they are more secure than passwords and can be used to verify a user's identity without needing to store a password in your database.
+When the OTP option is selected as an authentication strategy, users receive a one-time code to complete the sign-in process. OTPs are more secure than passwords, as they allow user verification without storing passwords in your database.
```

Comment:
"One of the OTP options" is correct - see the following paragraph that lists more than one OTP option.